### DISCUSSION TITLE
image gets converted to pink/purple ish colors when converting svg to png/jpg/svg

### DISCUSSION THREAD

**Discussion by tcvdh** [ImageMagick/ImageMagick#4426] [Help]
![2](https://user-images.githubusercontent.com/56931057/139528120-9ad4b67f-1420-4dc3-8809-f61de1b3b89c.png)
like this.... for some reason the colors get all weird when using "convert image.svg image.png"
the SVG looks good when opening in edge or chrome. but when converting with imagemagick convert or uploading the SVG to https://svgcheck.toolforge.org/index.php the image gets all purple ish
the original svg: https://svgshare.com/i/bb2.svg

**Comment by snibgo** [comment]
What version of IM, on what platform?

You haven't showed what you expect the output to look like. Why not?

Does the image look as you expect when you open the SVG with Inkscape? For me, with Inkscape v1.0, the circles are all black. Is this what you expect? If not, then the problem is either with Inkscape or your SVG.

If Inkscape doesn't do what you expect, there is little chance of IM doing what you expect.

Hint: I don't think hash then 8 hex digits is a valid SVG colour.

**Comment by urban-warrior** [comment]
If you remove the `ff` from your fill colors, ImageMagick correctly renders your image.  Its also works, if you use the internal SVG renderer.  Try
```
convert msvg:bb2.svg bb2.png
```
Your security policy must allow the MVG coder for this command to work.
